.gitignore

```diff
@@ -3,3 +3,4 @@
 /build
 /node_modules
 npm-debug.log*
+.idea
```

Comment:
You could add `.idea` to your local `.git/info/exclude`

Reply:
```suggestion
```

Reply:
Done